batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/IndexController.java

```diff
@@ -9,7 +9,7 @@
 
 import static com.lftechnology.batch7crud.constant.UrlConstants.URL_INDEX_PAGE;
 
-@WebServlet({ "/" })
+@WebServlet({ "/index" })
 
 public class IndexController extends HttpServlet {
   @Override
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)